config/locales/fr.yml

```
@@ -131,6 +131,8 @@ fr:
     instructeurs:
       dossiers:
         deleted_by_user: "Dossier supprimé par l'usager"
+      avis:
+        introduction_file_explaination: "Fichier d'introduction à l'avis fourni par"
```

Idéalement, apostrophes courbes :)

```suggestion
        introduction_file_explaination: "Fichier d’introduction à l’avis fourni par"
```

---

Et aussi, est-ce que plutôt que de parler de "fichier d'introduction à l'avis", qui m'a l'air un peu obscur, on ne pourrait pas parler de "Fichier joint à la demande d'avis" (et "Fichier joint à la réponse") ?